docs/cookbook/events_and_gestures.md

```diff
@@ -4,10 +4,10 @@ To add an event, use .on, to remove it it, use .off
 
 ```ruby
 # Simple example
-rmq(UIView).on(:tap){|sender| rmq(sender).hide}
+append(UIView).on(:tap){|sender| find(sender).hide}
 
```

Comment:
This should be find(UIView), not append. I guess this example is fine too, just noting it changed